packages/radix-ui-themes/src/components/card.css

```diff
@@ -99,44 +98,117 @@
 /* surface */
 
 .rt-Card.rt-variant-surface {
+  --inset-border-width: 1px;
+  --card-inner-border-radius: calc(var(--card-border-radius) - 1px);
```

Comment:
Should this use `--inset-border-width` instead of `1px` in the calc and also the next `border` rule?

Reply:
I had it like that and it felt so gnarly over a tiny 1px thing that I felt that keeping it readable and approachable was easier to grasp than to have it in a var. I am happy to go either way depending on what'd you prefer though